client/playwright.config.ts

```diff
@@ -128,7 +128,19 @@ export default defineConfig<ChromaticConfig>({
   projects: [
     {
       name: "chromium",
-      testMatch: "**/e2e/*.test.ts",
+      testMatch: [
+        "**/e2e/annotation.e2e.test.ts",
+        "**/e2e/cell-guide.test.ts",
+        "**/e2e/e2e.test.ts",
+      ],
+      use: {
+        ...devices["Desktop Chrome"],
+        userAgent: devices["Desktop Chrome"].userAgent + CZI_CHECKER,
+      },
```

Comment:
Cool pattern splitting the persistence runs into their own project so we can control launch config independently of the main chromium suite.